deck/views.py

```diff
@@ -30,7 +30,7 @@ class BaseEventView(object):
 
 class ListEvents(BaseEventView, ListView):
     template_name = 'event/event_list.html'
-    queryset = Event.objects.published_ones()
+    queryset = Event.objects.upcoming()
```

Comment:
We need to show the upcoming that are published in this page, with this code we will show all events, even if it is not published yet.

Sometimes the organizers already create the event but not want to allow people to send proposals yet.